nimble/host/src/ble_gatts.c

```diff
@@ -718,6 +718,8 @@ ble_gatts_clt_cfg_access_locked(struct ble_hs_conn *conn, uint16_t attr_handle,
             /* Successful writes get persisted for bonded connections. */
             if (conn->bhc_sec_state.bonded) {
                 out_cccd->peer_addr = conn->bhc_peer_addr;
+                out_cccd->peer_addr.type =
+                    ble_hs_misc_addr_type_to_id(conn->bhc_peer_addr.type);
```

Comment:
`ble_hs_misc_addr_type_to_id` is for own addr type

Reply:
@andrzej-kaczmarek thanks, fixed.